Integration test flaky test - Analysisd Tier 2

# Description 

We observed a flaky test due to wazuh-db is not running during the integration tests execution as part of this issue https://github.com/wazuh/wazuh/issues/35732

https://github.com/wazuh/wazuh/actions/runs/25046606797/attempts/1
```console
target_daemon = 'wazuh-db', running_condition = True, timeout = 10
extra_sockets = []

    def wait_expected_daemon_status(target_daemon=None, running_condition=True, timeout=10, extra_sockets=[]):
        """Wait until Wazuh daemon's status matches the expected one. If timeout is reached and the status didn't match,
           it raises a TimeoutError.
    
        Args:
            target_daemon (str, optional):  Wazuh daemon to check. Default `None`. None means all.
            running_condition (bool, optional): True if the daemon is expected to be running False
                if it is expected to be stopped. Default `True`.
            timeout (int, optional): Timeout value for the check. Default `10` seconds.
            extra_sockets (list, optional): Additional sockets to check. They may not be present in default configuration.
    
        Raises:
            TimeoutError: If the daemon status is wrong after timeout seconds.
        """
        condition_met = False
        start_time = time.time()
        elapsed_time = 0
    
        while elapsed_time < timeout and not condition_met:
            if sys.platform == WINDOWS:
                condition_met = check_if_process_is_running(WAZUH_AGENT_WIN) == running_condition
            else:
                control_status_output = subprocess.run([WAZUH_CONTROL_PATH, 'status'],
                                                       stdout=subprocess.PIPE).stdout.decode()
                condition_met = True
                for lines in control_status_output.splitlines():
                    daemon_status_tokens = lines.split()
                    current_daemon = daemon_status_tokens[0]
                    daemon_status = ' '.join(daemon_status_tokens[1:])
                    daemon_running = daemon_status == 'is running...'
                    if current_daemon == target_daemon or target_daemon is None:
                        if current_daemon in WAZUH_SOCKETS.keys():
                            socket_set = {path for path in WAZUH_SOCKETS[current_daemon]}
                        else:
                            socket_set = set()
                        # We remove optional sockets and add extra sockets to the set to check
                        socket_set.difference_update(WAZUH_OPTIONAL_SOCKETS)
                        socket_set.update(extra_sockets)
                        # Check specified socket/s status
                        for socket in socket_set:
                            if os.path.exists(socket) != running_condition:
                                condition_met = False
                        if daemon_running != running_condition:
                            condition_met = False
            if not condition_met:
                time.sleep(1)
            elapsed_time = time.time() - start_time
    
        if not condition_met:
>           raise TimeoutError(f"{target_daemon} does not meet condition: running = {running_condition}")
E           TimeoutError: wazuh-db does not meet condition: running = True

/usr/local/lib/python3.10/dist-packages/wazuh_testing/utils/services.py:233: TimeoutError
_________ ERROR at setup of test_validate_all_win32_alerts[Added1662] __________

```

[logs_66506255813.zip](https://github.com/user-attachments/files/27193735/logs_66506255813.zip)

## DoD 

- [ ] RCA of the failing test 
- [ ] Propose a fix if apply

--- a/src/wazuh_testing/utils/services.py
+++ b/src/wazuh_testing/utils/services.py
@@ -183,7 +183,7 @@ def check_all_daemon_status():
     return daemons_status
 
 
-def wait_expected_daemon_status(target_daemon=None, running_condition=True, timeout=10, extra_sockets=[]):
+def wait_expected_daemon_status(target_daemon=None, running_condition=True, timeout=30, extra_sockets=[]):
     """Wait until Wazuh daemon's status matches the expected one. If timeout is reached and the status didn't match,
        it raises a TimeoutError.
 
@@ -191,7 +191,7 @@ def wait_expected_daemon_status(target_daemon=None, running_condition=True, time
         target_daemon (str, optional):  Wazuh daemon to check. Default `None`. None means all.
         running_condition (bool, optional): True if the daemon is expected to be running False
             if it is expected to be stopped. Default `True`.
-        timeout (int, optional): Timeout value for the check. Default `10` seconds.
+        timeout (int, optional): Timeout value for the check. Default `30` seconds.
         extra_sockets (list, optional): Additional sockets to check. They may not be present in default configuration.
 
     Raises:

